### ISSUE TITLE
False positives of `prefer-lodash-method` with lodash-es

### ISSUE THREAD

**Issue by yuriy-yarosh** [wix/eslint-plugin-lodash#126]
So `prefer-lodash-method` detects lodash `_` import only with
```import * as _ from lodash```
and ignores
```import * as _ from lodash-es```

I'm using `lodash-es` primarily for tree shaking - reducing the size of the result bundles.

`lodash-es` is the es2015 modules build of the lodash4.

Atom screenshot:
![lodash-es](http://i.imgur.com/8UNRPaI.png)

**Comment by ganimomer** [comment]
Thanks for bringing this up! Added support for `lodash-es` in `v2.2.2`.

**Comment by yuriy-yarosh** [comment]
Thank you @ganimomer.
